README.md

```diff
@@ -1 +1,25 @@
-# hw01
+# Hello There
+
+My name is Allison Leon. I am a fourth year majoring in **Public Policy** with a minor in _Human Rights_. I am originally from *Chicago* and grew up in the **Back of the Yards neighborhood**. I have had experience with R in the past, as it is almost a requirement to familiarize yourself with it for the PubPol major. However, I always felt that those courses brushed through R too quickly, and I wanted to really understand and be able to use R for myself and future work. I am excited to be frustrated by this course, as this first assignment already proved!
+
+Here are a few more facts about me
+* I am Guatemalan
+* I have two younger sisters
+* I find it easiest to study while listening to [instrumental metal music](https://www.youtube.com/watch?v=wPSWsz2R6Xc&t=2857s&ab_channel=Dorkly)
+* I have a dog named Monch (pictured below)
+
```

Comment:
I really appreciate the usage of a list here, it makes everything organized and clear. And this link here is fun! Maybe I should try instrumental metal music next time too and see how it goes!